content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -97,38 +96,40 @@ By default all binaries are downloaded when you install Prisma CLI. There are ho
 
 ### <inlinecode>PRISMA_BINARIES_MIRROR</inlinecode>
 
-`PRISMA_BINARIES_MIRROR` can be used to specify a custom CDN endpoint (or any server) to download the engines binaries for the CLI/Client. The default value is `https://binaries.prisma.sh`.
+`PRISMA_BINARIES_MIRROR` can be used to specify a custom CDN (or server) endpoint to download the engines binaries for the CLI/Client. The default value is `https://binaries.prisma.sh`.
+
+See [`Navigating a corporate firewall`](../../concepts/overview/under-the-hood#navigating-a-corperate-firewall-with-prisma_binaries_mirror) for a conceptual overview of how to use this environment variable.
```

Comment:
"Navitationg a coporate firewall" would be an example for a problem to work around using that env var.